test/transform/ThemeTransform.test.js

```diff
@@ -49,21 +49,5 @@ describe('ThemeTransform', () => {
       const clazz = pagelib.ThemeTransform.CONSTRAINT.IMAGE_NO_BACKGROUND
       assert.ok(document.querySelector('img').classList.contains(clazz))
     })
-
```

Comment:
I was able to remove the tabular/non-tabular determination, so the test was no longer meaningful. If we can get things configured so we can add a couple of basic visual tests I'd like to circle back and add tests for each of these tricky theming areas - it's getting unwieldy even with the help of the theming demo to test and experiment with revisions in a safe way.